app/assets/javascripts/listings/templates/listing/_what_to_expect.html.slim

```diff
@@ -2,16 +2,16 @@ h4.h-caps-lined
   | What to Expect
 p.c-steel.t-small.margin-bottom
   ' Applicants will be contacted by the leasing agent
-  span ng-if="!$ctrl.parent.listingIsFirstComeFirstServe()"
+  span ng-if="!$ctrl.parent.isFirstComeFirstServe()"
```

Comment:
seems like we may need to pass the listing in here now, right?

Reply:
these are an instance of the component controllers using an implicit listing object from the component controller. i think this is a larger question about that pattern of using `@listing` as the default argument in component controller functions that pass a listing as an argument to a service function. let's add this topic to a list of general discussion points that we should discuss after your initial review of this PR